src/recording.jl

```diff
@@ -179,13 +179,18 @@ function Record(func, figlike, iter; kw_args...)
 end
 
 function Base.show(io::IO, ::MIME"text/html", vs::VideoStream)
+    scene = vs.screen.scene
+    if !(scene isa Scene)
```

Comment:
Why not `size(screen)`?

Reply:
seemed to return the framebuffer size in my test

Reply:
<img width="220" alt="image" src="https://github.com/user-attachments/assets/17ac799b-49ff-4b86-8316-061c3c95d219">